### ISSUE TITLE
Audio Crackling Skies Of Arcadia

### ISSUE THREAD

**Issue by jeremy27041980** [libretro/flycast#970]
Audio crackling when a character or a ship focus their aura. Low Pass Filter or DSP Issue ?

[](url) https://1drv.ms/u/s!AulQM9pAd1wehPJxDQeY-E6ti9vAsg

**Comment by inactive123** [comment]
Your system is likely too slow. Lower settings.

**Comment by jeremy27041980** [comment]
II forgot to say that I play the pal version in 50 Hertz and 50 fps. I have the same issue with the NTSC version in 60 hertz and 60 Fps
My system : i7 9900k and nvidia 2080 ti.

I can provide a save if you want to test.

**Comment by barbudreadmon** [comment]
There is a bunch of backport from upstream in https://github.com/libretro/flycast/pull/971, i believe one of them concern some audio issues in skies of arcadia. It'll be merged after some more testing.

**Comment by jeremy27041980** [comment]
I just updated the core in retroarch and still have the crackle for that specific sound. No problem for the other game sounds

**Comment by barbudreadmon** [comment]
ok, thanks for the heads-up, do you know if it happens with standalone builds from https://flyinghead.github.io/flycast-builds/ ?

**Comment by jeremy27041980** [comment]
Yes, it happens with standalone builds. I just try it. Here my save.
[skies of Arcadia.zip](https://github.com/libretro/flycast/files/5408809/skies.of.Arcadia.zip)

**Comment by barbudreadmon** [comment]
I guess it's a duplicate of https://github.com/flyinghead/flycast/issues/108 then, the issue doesn't seem fully solved.

**Comment by jeremy27041980** [comment]
I have also noticed some static noise in the game, but it is very rare. If you take my save, you can hear it when the pillars fall.
